### DISCUSSION TITLE
Does deleting account also remove the auto-synced collections?

### DISCUSSION THREAD

**Discussion by gayathrirv** [Kong/insomnia#6604] [Help] [answered]
I would like to know whether deleting an account would also remove the collections synced to the cloud.
After the upgrade to 8.x, I created an account yesterday and before I could comprehend what was happening, my collections were synced to the cloud. As it contained private data, I was not happy with the upload and I immediately deleted the account. However, it is not clear if this also removed the uploaded collections. Can someone please help me with this? 

This is the message I received after deleting my account and I was the only collaborator in this workspace. 
> We want to inform you that the owner of the Personal Workspace organization has removed you as a collaborator.

@subnetmarco could you please clarify this?

**Comment by subnetmarco** [comment, marked as answer]
Yes, deleting an account will permanently delete all data.

**Reply by gayathrirv** [reply]
Thank you for the clarification! 
Feedback:
> We want to inform you that the owner of the Personal Workspace organization has removed you as a collaborator.

This message after account deletion is not very informative regarding the workspace/collections owned by the deleted user. It would be nice to have that information as well.
